2021-05-23 11:51:06,239 INFO    MainThread:43900 [wandb_setup.py:_flush():69] setting env: {}
2021-05-23 11:51:06,239 INFO    MainThread:43900 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-23 11:51:06,239 INFO    MainThread:43900 [wandb_init.py:_log_setup():336] Logging user logs to /home/ubuntu/data/matengfei/kg-mtl/wandb/run-20210523_115106-2enb9vw0/logs/debug.log
2021-05-23 11:51:06,239 INFO    MainThread:43900 [wandb_init.py:_log_setup():337] Logging internal logs to /home/ubuntu/data/matengfei/kg-mtl/wandb/run-20210523_115106-2enb9vw0/logs/debug-internal.log
2021-05-23 11:51:06,240 INFO    MainThread:43900 [wandb_init.py:init():369] calling init triggers
2021-05-23 11:51:06,240 INFO    MainThread:43900 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'dropout': 0.2, 'n_hidden': 500, 'gpu': 1, 'lr_pre': 0.01, 'lr_dti': 0.001, 'n_bases': 4, 'sample_size': 4, 'n_layers': 2, 'n_epochs': 100, 'regularization': 0.01, 'grad_norm': 1.0, 'graph_split_size': 0.5, 'negative_sample': 10, 'edge_sampler': 'neighbor', 'graph_batch_size': 40000, 'rgcn_epochs': 0, 'loss_lamda': 0.75, 'cpi_dataset': 'human', 'dti_dataset': 'drugcentral', 'shared_unit_num': 1, 'embedd_dim': 128, 'loss_mode': 'single'}
2021-05-23 11:51:06,240 INFO    MainThread:43900 [wandb_init.py:init():418] starting backend
2021-05-23 11:51:06,240 INFO    MainThread:43900 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-23 11:51:06,246 INFO    MainThread:43900 [backend.py:ensure_launched():132] starting backend process...
2021-05-23 11:51:06,250 INFO    MainThread:43900 [backend.py:ensure_launched():137] started backend process with pid: 43909
2021-05-23 11:51:06,251 INFO    MainThread:43900 [wandb_init.py:init():423] backend started and connected
2021-05-23 11:51:06,254 INFO    MainThread:43900 [wandb_init.py:init():465] updated telemetry
2021-05-23 11:51:06,255 INFO    MainThread:43900 [wandb_init.py:init():484] communicating current version
2021-05-23 11:51:06,696 INFO    MainThread:43900 [wandb_init.py:init():489] got version response 
2021-05-23 11:51:06,696 INFO    MainThread:43900 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-23 11:51:06,897 INFO    MainThread:43900 [wandb_init.py:init():522] starting run threads in backend
2021-05-23 11:51:07,157 WARNING MainThread:43900 [wandb_init.py:init():764] interrupted
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 744, in init
    run = wi.init()
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 525, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 799, in communicate_run_start
    result = self._communicate(rec)
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 537, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/ubuntu/anaconda3/envs/make_cpi/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
